README.md

```diff
@@ -41,13 +41,13 @@ The following software requirements are necessary to run the Bold Reports Enterp
 
 | Tags               | OS Version    | Last Modified |
 | -------------      | ------------- | ------------- |
-| `5.3.8`           | Ubuntu 20.04  (amd64)    | 26/10/2023 |
-| `5.3.8_alpine`    | Alpine 3.13  (amd64)  | 26/10/2023 |
-| `5.3.8_debian`     | Debian 10  (amd64,arm64)        | 26/10/2023 |
-|`5.3.8_arm64`|Debian 10 (arm64)|26/10/2023 |
-|`5.3.8_ubuntu_arm64`| Ubuntu 20.04  (arm64)        | 26/10/2023 |
+| `5.4.20`           | Ubuntu 20.04  (amd64)    | 26/10/2023 |
```

Comment:
updated today's date in the last modified section

Reply:
updated